src/forscape_parser.cpp

```diff
@@ -2053,12 +2070,11 @@ ParseNode Parser::error(ErrorCode code) alloc_except {
 }
 
 ParseNode Parser::error(ErrorCode code, const Typeset::Selection& c) alloc_except {
-    if(error_node == NONE){
+    if(noErrors()){
         error_stream.fail(c, code);
-        error_node = parse_tree.addTerminal(OP_ERROR, Typeset::Selection(c));
     }
```

Comment:
The goal here is to limit error message spam, but that shouldn't be the parser's concern